tests/validation/tests/v3_api/test_import_rke2_cluster.py

```diff
@@ -79,7 +90,16 @@ def create_rke2_multiple_control_cluster(cluster_type, cluster_version):
     tf_dir = DATA_SUBDIR + "/" + "terraform/rke2/master"
     keyPath = os.path.abspath('.') + '/.ssh/' + AWS_SSH_KEY_NAME
     os.chmod(keyPath, 0o400)
-    no_of_servers = int(RANCHER_RKE2_NO_OF_SERVER_NODES) - 1
+
+    split_roles = str(RKE2_SPLIT_ROLES).lower()
+    if split_roles == "true":
+        no_of_servers = int(RANCHER_RKE2_NO_OF_SERVER_NODES) + \
+            int(RKE2_ETCD_ONLY_NODES) + int(RKE2_ETCD_CP_NODES) + \
+            int(RKE2_ETCD_WORKER_NODES) + int(RKE2_CP_ONLY_NODES) + \
+            int(RKE2_CP_WORKER_NODES) - 1
+
+    else:
+        no_of_servers = int(RANCHER_RKE2_NO_OF_SERVER_NODES) - 1
 
```

Comment:
Nit-picky: Can we add a check that if any of those values (RKE2_ETCD_ONLY_NODES, RKE2_ETCD_CP_NODES,etc) are > 0, their corresponding number should exist in RKE2_ROLE_ORDER?

Reply:
Most recent commit (squashed) should resolve this. Great suggestion!

Reply:
LGTM Thanks!